js/front/dataset/resource-modal.vue

```diff
@@ -7,6 +7,8 @@
 
         <dl class="dl-horizontal dl-wide">
           <dt>{{ _('URL') }}</dt>
+          <dd><a :href="resource.contentUrl" @click="onClick">{{resource.contentUrl}}</a></dd>
+          <dt>{{ _('Permalink') }}</dt>
```

Comment:
To me `Latest URL` seems dynamic, meaning that it's changing over the time.
Whereas permalink is totally defined: https://en.wikipedia.org/wiki/Permalink